src/acl_storage.rs

```diff
@@ -0,0 +1,140 @@
+// Copyright 2015-2020 Parity Technologies (UK) Ltd.
+// This file is part of Parity Ethereum.
+
+// Parity Ethereum is free software: you can redistribute it and/or modify
```

Comment:
Probably we want to change "Parity Ethereum" to "Parity Secret Store" everywhere?

Reply:
Good question. Also for @dvdplm and @folsen

Reply:
License will have to stay GPL e.g. because of the GPL dependencies like `kvdb`, but I agree we want to change `Parity Ethereum` to `Parity Secret Store` here.

Reply:
renamed

Reply:
@grbIzl It is now "Parity Ethereum Secret Store" :) Let's rename to "Parity Secret Store" :)

Reply:
ok